src/pages/HomePage.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const HomePage: React.FC = () => {
+  return (
+    <>
+      <h1 className="title">Home Page</h1>
+    </>
+  );
+};
```

Comment:
```suggestion
export const HomePage: React.FC = () => (
  <h1 className="title">Home Page</h1>
);
```

Reply:
If your components don't have logic for rendering, will be better for a short render format.